heartbeat/awseip

```diff
@@ -182,9 +182,8 @@ awseip_start() {
 awseip_stop() {
     awseip_monitor || return $OCF_SUCCESS
 
-    ASSOCIATION_ID=$($AWSCLI --profile $OCF_RESKEY_profile --output json ec2 describe-addresses \
-                         --allocation-id ${ALLOCATION_ID} | grep -m 1 "AssociationId" | awk -F'"' '{print$4}')
-    $AWSCLI --profile $OCF_RESKEY_profile ec2 disassociate-address  \
+    $AWSCLI --profile ${OCF_RESKEY_profile} \
+        ec2 disassociate-address \
         --association-id ${ASSOCIATION_ID}
```

Comment:
@oalbrigt please consider to merge this commit to avoid `ASSOCIATION_ID` become `empty` while command failure, I think it would be better to move allocation_id check to `awseip_monitor()` instead.

Reply:
It's not a check. It's only used to disassociate the address, which is why it's defined in stop(). See line 187.

Reply:
@oalbrigt 
imagine the Elastic IP have been disassociated with the node, it will lead to a empty string for the variable `ASSOCIATION_ID`, and `awscli` will complain that there's a no value error on ` --association-id ${ASSOCIATION_ID}` at line 187, isn't it?

Reply:
I still insist to remove the existence check of `${ASSOCIATION_ID}` to make the code cleaner

line 201 ~ 204 in `awseip_monitor()` have already checked the existence of `${ASSOCIATION_ID}`
if it doesn't exist, `awseip_monitor()` would return a non-zero value, then trigger `... || return $OCF_SUCCESS` at line 183

Reply:
That is ALLOCATION_ID, which is defined by the allocation_id-parameter.

ASSOCIATION_ID changes each time the IP is associated with a node, which is why we get the information from AWS.

Reply:
okay, got your point, reverting back.

Reply:
@oalbrigt reverted, and re-commit. please review.